saas_public/controllers/saas_public.py

```diff
@@ -5,13 +5,14 @@
 
 
 class SaaSPublicController(Controller):
-    @route('/saas_public/<model("saas.template"):template_id>/create-fast-build', type='http', auth='public')
+    @route('/saas_public/<int:template_id>/create-fast-build', type='http', auth='public')
     def create_fast_build(self, template_id, **kwargs):
         if not kwargs:
             kwargs = {}
-        if template_id and template_id.sudo().public_access:
-            template_operator_id = template_id.sudo().operator_ids.random_ready_operator()
-            build = template_operator_id.sudo().create_db(kwargs, with_delay=False)
+        template = request.env['saas.template'].browse(template_id).sudo()
+        if template and template.public_access:
```

Comment:
make ``else`` block and return something

Reply:
It is here https://github.com/it-projects-llc/saas-addons/blob/f77dbd99a2fca88065704ad089359a41d7bff755/saas_public/controllers/saas_public.py#L19